recipe/meta.yaml

```diff
@@ -28,8 +28,7 @@ requirements:
     - pkg-config
     - pthread-stubs         # [linux]
     - xcb-proto             # [linux]
-    - xorg-renderproto      # [linux]
-    - xorg-xproto           # [linux]
+    - xorg-xorgproto        # [linux]
```

Comment:
These xorg requirements in the build section are dubious, are they truely necessary?

Reply:
I can't really speak to the needs of this recipe or the details of the new xorg setup...

Reply:
yeah once the recipe is building, lets try to slim these down before we merge on main